src/testsets.jl

```diff
@@ -339,9 +339,12 @@ function display_reporting_testset(ts::ReportingTestSet)
     try
         # Finish each of the results of the top level testset, to mimick the
         # output from Pkg.test()
-        finish.(ts_default.results)
-    catch TestSetException
+        for r in ts_default.results
+            r isa AbstractTestSet && finish(r)
+        end
+    catch err
```

Comment:
Should the try-catch be inside the `for` loop?

Reply:
Although that would display more results, it's not how `Pkg.test` would display the results (that is if first testset of a package fails then the second one wouldn't be displayed as an error is thrown when the first testset `finish`es. So will keep it as is for now